src/HonzaBotner.Discord.Services/SlashCommands/BotSCommands.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Threading.Tasks;
+using DSharpPlus;
+using DSharpPlus.Entities;
+using DSharpPlus.SlashCommands;
+using DSharpPlus.SlashCommands.Attributes;
+using HonzaBotner.Discord.Services.Options;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace HonzaBotner.Discord.Services.SlashCommands
+{
+    [SlashCommandGroup("bot", "Příkazy pro získání informací o botovi a jeho základní nastavení")]
+    public class BotSCommands : ApplicationCommandModule
+    {
+        private readonly ILogger<BotSCommands> _logger;
+        private readonly InfoOptions _options;
+
+        public BotSCommands(ILogger<BotSCommands> logger, IOptions<InfoOptions> options)
+        {
+            _logger = logger;
+            _options = options.Value;
+        }
+
+        [SlashCommand("activity", "Aktivita kterou bot zobrazí na svém profilu", false)]
+        [SlashRequireUserPermissions(Permissions.ManageGuild, false)]
+        public async Task ActivityAsync(InteractionContext ctx,
+            [Option("Aktivita", "Nazev vykonavane aktivity")] string message,
+            [Option("Druh", "Druh aktivity ktera se zobrazi u bota")]
+            [Choice("Competing", "Competing")]
+            [Choice("ListeningTo", "ListeningTo")]
+            [Choice("Playing", "Playing")]
+            [Choice("Watching", "Watching")]
+            string slashActivityType = "Competing")
+        {
```

Comment:
nit: Formating, pls. Each parameter is on a separate line.

Reply:
Recommended way:
![image](https://user-images.githubusercontent.com/29132060/136084046-68db862a-8951-4436-b01e-2fd9a679c84b.png)

It can really add up quickly. Do you really want to put it all on 1 line?

Reply:
Well, you get less git diff for lines that were not changed and it even looks a bit more separated.
So yes, I would go for that.